### ISSUE TITLE
Errors Encountered in building liboorb

### ISSUE THREAD

**Issue by OrbitalMechanic** [oorb/oorb#41]
I'm attempting to install orb on a MacPro (late 2013) desktop machine running Mac OS X ver. 10.12.3 (Sierra). I'm using fortran under gcc version 7.1.0 (obtained via Homebrew). 

The problem I'm having is that the build cannot determine the type for the following functions:

                                         containsSampledPDF     getPDFValues

Furthermore, the following statement have syntax errors:

                                         obsy_ccoord=obsy_ccoord)
                                         pdfs_arr=pdfs_arr)
                                         sor_generat_offset=sor_genwin_offset)

All of these error are captured in the text file attached to this note.

Please advise.

Sam Dupree.

[liboorb_build.txt](https://github.com/oorb/oorb/files/1609614/liboorb_build.txt)

**Comment by Siltala** [comment]
Hi,

I can confirm the bug. The problem with containsSampledPDF and getPDFValues seems to have been caused by 76c746989858a119cbf57f9c6a57b7b8b17ae157 where Mikael renamed these fuctions but apparently forgot to update liboorb.f90 to reflect this, so this has probably been broken for almost a year now (!). That's easy enough to fix but still leaves the second part of the issue, which I'm not sure how to fix, at least based on the quick look that I did. Mikael will hopefully chime in here on that.
